docs/en/enterprise-edition/policy-reference/azure-policies/azure-networking-policies/bc-azure-239.adoc

```diff
@@ -0,0 +1,62 @@
+== Azure Machine learning workspace is not configured with private endpoint
+
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+| a8e6f787-f937-42eb-9f71-ffc4cb17ba8a
+
+|Checkov ID
+| https://github.com/bridgecrewio/checkov/blob/main/checkov/arm/checks/resource/AzureMLWorkspacePrivateEndpoint.py[CKV_AZURE_239]
+
+|Severity
+|MEDIUM
+
+|Subtype
+|Build
+
+|Frameworks
+|ARM,Bicep
+
+|===
+
+=== Description
+
+This policy is checking to confirm that an Azure Machine learning workspace is not set up with a private endpoint. Using a private endpoint can pose a risk as it exposes the system to access via the private network only, hence limiting the accessibility, scalability, and flexibility of the system. Private endpoints further risk a single point of failure, which can directly affect the availability of the whole application or service.
+
+=== Fix - Buildtime
+
+*ARM*
+
+* *Resource:* Microsoft.MachineLearningServices/workspaces
+* *Arguments:* managedNetwork.outboundRules
+
+To fix the issue, you should configure the Azure Machine Learning workspace to use a private endpoint. This can be done by adding an outboundRule with type `PrivateEndpoint`.
```

Comment:
Configure the Azure Machine Learning workspace to use a private endpoint. To achieve this, add an outboundRule with the type set to `PrivateEndpoint`. This effectively directs traffic to the workspace solely through the secure private network connection.